src/com/sap/piper/API.java

```diff
@@ -0,0 +1,19 @@
+package com.sap.piper;
```

Comment:
On my side it does not work when renaming the `src/com/sap/piper/API.java` file to `*.groovy`. This ends up in compilation errors. I did not invest into this since it does - to my understanding - not make a difference since groovy is a superset of java. Each plain java code can be executed by groovy. 

> I'm not totally sure if this is any issue when used in Jenkins Pipelines?

Just for the records: the annotation is not intended for being used in the pipeline script (aka Jenkinsfile).